cmd/protoc-gen-gorums/gengorums/gorums.go

```diff
@@ -192,6 +193,12 @@ func (c *callTypeInfo) deriveCallType(m *protogen.Method) *callTypeInfo {
 	return c
 }
 
+// callTypeName returns the name of the call type option with the prefix removed
+func callTypeName(ext protoreflect.ExtensionDescriptor, prefixLen int) string {
```

Comment:
If you make this take `*protoimpl.ExtensionInfo`, you can do the call to `TypeDescriptor()` her instead.

Reply:
Cool, I didn't realize that there was a non-internal alias to that type.